ts/lib/auth/googleauth.ts

```diff
@@ -268,7 +275,8 @@ export default class GoogleAuth {
    * @return {boolean} Returns true if the callback has been executed; false otherwise.
    * @api private
    */
-  public _tryGetApplicationCredentialsFromWellKnownFile(opt_callback?) {
+  public _tryGetApplicationCredentialsFromWellKnownFile(
+      opt_callback?: (err: Error, result: any) => void) {
```

Comment:
Added the ? to the result.  On dumping the any(s) here - most of the places where I left an any, it was because the data was pulled from a JSON.parse(...) where I'm not 100% sure about the contents.  I want to create a follow up issue to lock down the types even more if I can :)